topics/home.md

```diff
@@ -3,38 +3,18 @@
 
 ## Get started
 
-* If you're already familiar with the Kotlin language and want to try out Kotlin Multiplatform Mobile, check out our [getting started page](getting-started.md).
-* If you're new to Kotlin, read our [language documentation](https://kotlinlang.org/docs/reference/) or play with Kotlin on the [Playground](https://play.kotlinlang.org/koans/overview). Both of these are good places to learn about the basic language concepts.
+* If you're already familiar with the Kotlin language and want to try out KMM, [get started with KMM](getting-started.md).
+* If you're new to Kotlin, [get started with Kotlin](https://kotlinlang.org/docs/tutorials/getting-started.html) or [play with Kotlin online](https://play.kotlinlang.org/koans/overview). 
```

Comment:
I'm afraid it might be confusing for new users: what's the difference between "get started with Kotlin" and "play with Kotlin online"?

Reply:
Let's add _get started with Kotlin in an IDE_. Is this clearer?

Reply:
Yes, that sounds better!